cmake/findDependencies.cmake

```diff
@@ -46,6 +46,10 @@ if( EXIV2_ENABLE_PNG )
     find_package( ZLIB REQUIRED )
 endif( )
 
+if( EXIV2_ENABLE_BMFF )
```

Comment:
[question] Do you always want to find the need library whenever BMFF is enabled? Are the two things tight together? Would it make sense to have a new CMake option for the support of the new library?

Reply:
We could introduce an external switch as well... It is not a hard requirement for BMFF since it is (for now) specific only to JPEG XL, so I wanted to keep it simple.

Reply:
I think you have a point - I'll make it fully controllable so one has the option not to depend on it and still have (most of) the BMFF support.